### PR TITLE
Solve misc issues reported in Transifex

### PR DESCRIPTION
**Pull request by DelazJ**
It happens that translators finds issues in strings or have hard time to understand how they could translate some strings (e.g. contexts are generally poorly provided, and words do not always have a single translation - devs should do an effort on that topic) so... people comment and prevent the others (https://www.transifex.com/qgis/QGIS/translate/#en_US/$/114604231?q=issue%3Aopen -- _replace en_US with your language_).
This PR addresses some of their comments.

PS: I have no idea how the Oracle Spatial string is supposed to render and if someone knows how to fill the reference relation's one...

### CODE REVIEW

File: python/plugins/db_manager/db_plugins/oracle/info_model.py [hunk 1]
```diff
@@ -512,8 +511,7 @@ def getMViewInfo(self):
                     values[9]))
         tbl.append((QApplication.translate(
             "DBManagerPlugin",
-            (u'<a href="action:mview/refresh">Refresh the materializ'
-             u'ed view</a>')),
+            (u'<a href="action:mview/refresh">Refresh the materialized view</a>')),
```
Comment (nyalldawson) [inline python/plugins/db_manager/db_plugins/oracle/info_model.py:514]: Wouldn't it be cleaner to separate the translated part of this string from the html formatting surrounding it?

File: src/ui/editorwidgets/qgsrelationreferenceconfigdlgbase.ui [hunk 2]
```diff
@@ -122,7 +122,7 @@
       <item row="5" column="0" colspan="3">
        <widget class="QCheckBox" name="mCbxChainFilters">
         <property name="toolTip">
-         <string>When activated, the filters will restrict the choices of fields to options that are </string>
+         <string>When activated, the filters will restrict the choices of fields to options that are (MISSING WORDS?)</string>
```
Comment (nyalldawson) [inline src/ui/editorwidgets/qgsrelationreferenceconfigdlgbase.ui:125]: Just remove?

Comment (agiudiceandrea) [reply src/ui/editorwidgets/qgsrelationreferenceconfigdlgbase.ui:125]: Maybe @m-kuhn knows (https://github.com/qgis/QGIS/commit/733b7c9c8d83062c053a13743a7a2b0afc5fce26).

**Comment by agiudiceandrea** [comment]
@DelazJ have you seen https://lists.osgeo.org/pipermail/qgis-developer/2022-October/065176.html and the thread at https://github.com/qgis/QGIS/commit/1992600de3034915238c21665ad7af6104c468eb#commitcomment-88113964?

File: python/plugins/db_manager/db_plugins/oracle/info_model.py (hunk 1 above)
Comment (DelazJ) [reply python/plugins/db_manager/db_plugins/oracle/info_model.py:514]: That would result in something like this? Correct?
```py
        tbl.append((u'<a href="action:mview/refresh">{0}</a>'.format(
            QApplication.translate("DBManagerPlugin", "Refresh the materialized view")),
            u""))
```
For information, I couldn't find this string neither in transifex nor qgis_en.ts file 🤔 
image

File: src/ui/editorwidgets/qgsrelationreferenceconfigdlgbase.ui (hunk 2 above)
Comment (DelazJ) [reply src/ui/editorwidgets/qgsrelationreferenceconfigdlgbase.ui:125]: @m-kuhn ?

**Comment by DelazJ** [comment]
@agiudiceandrea Yes a quite misleading issue. Good catch. I mention it in Transifex, hoping that translations will be fixed accordingly.

File: src/ui/editorwidgets/qgsrelationreferenceconfigdlgbase.ui (hunk 2 above)
Comment (m-kuhn) [reply src/ui/editorwidgets/qgsrelationreferenceconfigdlgbase.ui:125]: https://github.com/qgis/QGIS/commit/733b7c9c8d83062c053a13743a7a2b0afc5fce26#diff-c90e64112a47b9f71d3867e4209267e6e23abbaea6b5201a6b8fbff07fb131b1R102

> Chained filters restrict the option of subsequent filters based on the selection of a previous filter.

I think we want this

File: python/plugins/db_manager/db_plugins/oracle/info_model.py (hunk 1 above)
Comment (nyalldawson) [reply python/plugins/db_manager/db_plugins/oracle/info_model.py:514]: Yes, that's correct. We don't need the "u" prefixes anymore either.

**Review by m-kuhn — APPROVED** [review]

**Pull request merged**